game_logic subfolder (#237)
Akuli/catris: README.md

Unlike `Client` objects, the `ClientInfo` objects can't be used to send or receive;
they are purely information for game logic and other clients.

A lobby owns `GameWrapper`s, which take care of the timing and async aspects of a game.

> It has an underlying `Game` object, from `game_logic/game.rs`,

Author: ```suggestion
Each `GameWrapper` has an underlying `Game` object from `game_logic/game.rs`,
```

> and it is purely game logic just like the rest of the `game_logic/` folder:
> there's no async functions or IO inside `game_logic/`.

Author: ```suggestion
and like the rest of the `game_logic/` folder, it's purely game logic without any async or IO.
```
